src/internal/polyfills/Array.from.js

```diff
@@ -0,0 +1,24 @@
+import isIterable from '../../isIterable';
+import isUndefined from '../../isUndefined';
+
+const copyArray = (items, mapFn, thisArg) => {
+  if (isUndefined(mapFn)) return [...items];
+  if (isUndefined(thisArg)) return [...items].map(mapFn);
+  return [...items].map(mapFn).bind(thisArg);
```

Comment:
I don't think you're correctly binding the `thisArg`. It should be more like:
```js
const copyArray = (items, mapFn, thisArg) => {
  const boundMapFn = isNotUndefined(mapFn) ? bind(mapFn, thisArg) : mapFn;

  return isUndefined(boundMapFn) ? [...items]  :  [...items].map(boundMapFn);
};
```
`bind` function is from ramda